bin/local.js

```diff
@@ -76,6 +71,7 @@ function startFileWatch() {
     watcher.watch(config.sourceLocation, {
         ignored: ignored,
         persistent: true,
+        followSymlinks: false,
```

Comment:
Might expose this in a config, but since we've got symlinks like crazy in most of our apps, I've false'd it for now.